spec/requests/devices_spec.rb

```diff
@@ -32,13 +32,14 @@
       end
 
       it "does not increase device count" do
-        post "/users/devices", params: params
+        post devices_path, params: params
         expect(user.devices.count).to eq(0)
       end
 
       it "returns an error" do
-        post "/users/devices", params: params
-        expect(response.status).to eq(400)
+        post devices_path, params: params
+
+        expect(response).to have_http_status(:unprocessable_entity)
```

Comment:
this is a change in behavior. Before it was returning 400, now 422 as I merged https://github.com/forem/forem/pull/13647/files#diff-cbe5e271acea305fbca0724e50460c97184a405e1916b0f9893c24e2b517ae93R8

@fdoxyz let me know if I should keep 400, in case the iOS app makes a distinction between the two